CROWDIN.md

```diff
@@ -0,0 +1,62 @@
+# Crowdin Translator How-To
+
+Stellar Developer Docs use [Crowdin](https://crowdin.com/) for localization. This document provides steps and resources for managing Stellar Docusaurus Crowdin translations as a Crowdin user with a Translator role.
+
+---
+
+## Overview
+
+As new English source files are added to the Stellar Docusaurus Documentation GitHub repository, they are automatically uploaded to Crowdin. From there, an automated AI translation is run on the files to translate them into other target languages (currently only Spanish at the time of writing).
+
+In cases where the AI translation is inaccurate, a user with the Translator role will need to update and edit the translations manually.
+
+---
+
+## Translator User
+
+To get started, you need to set up a Crowdin account and be added to the Stellar Documents Crowdin project. This involves creating a Crowdin user account and requesting access to the project from a project owner. 
+
+For more details, see the following documentation: 
+[Create an Account](https://support.crowdin.com/for-translators/#create-an-account)
+
+---
+
+## Editor
+
+As a translator, you will have access to the editor, where you can make changes to the content. 
+
+**Note:** Crowdin uses the term "strings" for content being translated.
+
+You can access all translated documents via the Crowdin dashboard. From there, you can view each document in the editor. 
+
+For more details on working within the editor, see the following documentation: 
+[Working in the Editor](https://support.crowdin.com/for-translators/#working-in-the-editor) 
+
+For a more in-depth look, see: 
+[Online Editor Guide](https://support.crowdin.com/online-editor/)
+
+---
+
+## Glossary
+
+For common words or phrases, it is good practice to add them to the glossary. The glossary helps the AI translator provide more accurate translations. 
+
+Examples include: 
+- Words or phrases that need to be translated in a specific manner. 
+- Words or phrases that should not be translated at all. 
+
+For more information, see the following documentation: 
+[Glossary](https://support.crowdin.com/glossary/)
+
+---
+
+## Approving Translations
+
+As documents are translated, they will be assigned a translation percentage highlighted in blue. 
+
+To manage files that have been checked or proofread, it is recommended to mark them as approved. Approved files will display an approved percentage highlighted in green. 
+
+**Note:** All translations are synced with the Stellar Docusaurus Documentation regardless of whether they have been marked as approved. Approval is purely for managing translation files that have been checked. 
```

Comment:
this is probably something we should think about as people begin contributing. The risk of chatgpt doing "something" weird with the markdown/react syntax and causing build problems is definitely non-zero. Having an "approved" state that we're confident will build and work isn't necessarily a bad idea

Reply:
Yes, let me edit this note actually. It's a good idea to have an approval process. Also crowdin is set up with roles to enable this review flow